content/ch/starstreak.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Star Streaks
+layout: starstreak
+---
```

Comment:
We can remove this file (we'll add Chinese-language in a future PR using the go-18n for Hugo approach)